libraries/lib-stretching-sequence/AudioSegmentFactory.cpp

```diff
@@ -38,10 +38,11 @@ AudioSegmentFactory::CreateAudioSegmentSequence(
 std::vector<std::shared_ptr<AudioSegment>>
 AudioSegmentFactory::CreateAudioSegmentSequenceForward(double t0) const
 {
-   ClipConstHolders sortedClips { mClips };
+   auto sortedClips = mClips;
```

Comment:
I really dislike this entire commit -- adding mutability.

Let it stand, but I may explain the alternative with another suggested commit.